File: Go/math.go
package main

import (
	"fmt"
	"math"
)

func main() {
	// Add two numbers together and then return the sum.
	fmt.Println(addNumbers(1, 2))
	// Subtract two numbers and then return the difference.
	fmt.Println(subtractNumbers(4, 2))
	// Multiply two numbers and then return the product.
	fmt.Println(multiplyNumbers(4, 2))
	// Divide two numbers and then return the quotient.
	fmt.Println(divideNumbers(4, 2))
	// Modulo two numbers and then return the remainder.
	fmt.Println(moduloNumbers(4, 2))
	// Get the square root of a number and then return the square root.
	fmt.Println(squareRootNumbers(4))
}

// Add two numbers together and then return the sum.
func addNumbers(primary int, secondary int) int {
	return primary + secondary
}

// Subtract two numbers and then return the difference.
func subtractNumbers(primary int, secondary int) int {
	return primary - secondary
}

// Multiply two numbers and then return the product.
func multiplyNumbers(primary int, secondary int) int {
	return primary * secondary
}

// Divide two numbers and then return the quotient.
func divideNumbers(primary int, secondary int) int {
	return primary / secondary
}

// Modulo two numbers and then return the remainder.
func moduloNumbers(primary int, secondary int) int {
	return primary % secondary
}

// Get the square root of a number and then return the square root.
func squareRootNumbers(primary float64) float64 {
	return math.Sqrt(primary)
}

File: C++/math.cpp
#include <iostream>

// Take in two numbers and return the sum
int add_numbers(int a, int b) {
    return a + b;
}

// Take in two numbers and return the difference
int subtract_numbers(int a, int b) {
    return a - b;
}

// Take in two numbers and return the product
int multiply_numbers(int a, int b) {
    return a * b;
}

// Take in two numbers and devide them.
int divide_numbers(int a, int b) {
    return a / b;
}

int main() {
    // Add two numbers
    printf("%d\n", add_numbers(0, 1));
    // Subtract two numbers
    printf("%d\n", subtract_numbers(3, 1));
    // Multiply two numbers
    printf("%d\n", multiply_numbers(1, 3));
    // Devide two numbers
    printf("%d\n", divide_numbers(12, 3));
    return 0;
}
